dev-util/web_page_replay_go/web_page_replay_go-20210916.ebuild

```diff
@@ -0,0 +1,81 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+inherit go-module
+
+EGIT_COMMIT=57c353ca10632889d961e58f8acc5ca671c2c17b
+
+EGO_SUM=(
+	"github.com/BurntSushi/toml v0.3.1/go.mod"
+	"github.com/cpuguy83/go-md2man/v2 v2.0.0-20190314233015-f79a8a8ca69d"
+	"github.com/cpuguy83/go-md2man/v2 v2.0.0-20190314233015-f79a8a8ca69d/go.mod"
+	"github.com/kylelemons/godebug v1.1.0"
+	"github.com/kylelemons/godebug v1.1.0/go.mod"
+	"github.com/pmezard/go-difflib v1.0.0"
+	"github.com/pmezard/go-difflib v1.0.0/go.mod"
+	"github.com/russross/blackfriday/v2 v2.0.1"
+	"github.com/russross/blackfriday/v2 v2.0.1/go.mod"
+	"github.com/shurcooL/sanitized_anchor_name v1.0.0"
+	"github.com/shurcooL/sanitized_anchor_name v1.0.0/go.mod"
+	"github.com/urfave/cli v1.22.4"
+	"github.com/urfave/cli v1.22.4/go.mod"
+	"golang.org/x/crypto v0.0.0-20190308221718-c2843e01d9a2/go.mod"
+	"golang.org/x/net v0.0.0-20200602114024-627f9648deb9"
+	"golang.org/x/net v0.0.0-20200602114024-627f9648deb9/go.mod"
+	"golang.org/x/sys v0.0.0-20190215142949-d0b11bdaac8a/go.mod"
+	"golang.org/x/sys v0.0.0-20200323222414-85ca7c5b95cd/go.mod"
+	"golang.org/x/text v0.3.0"
+	"golang.org/x/text v0.3.0/go.mod"
+	"gopkg.in/check.v1 v0.0.0-20161208181325-20d25e280405/go.mod"
+	"gopkg.in/yaml.v2 v2.2.2/go.mod"
+	)
+go-module_set_globals
+
+SRC_URI="https://chromium.googlesource.com/catapult/+archive/${EGIT_COMMIT}.tar.gz -> catapult-${PV}.tar.gz
+	${EGO_SUM_SRC_URI}"
+
+DESCRIPTION="A performance testing tool for recording and replaying web pages"
+HOMEPAGE="https://chromium.googlesource.com/catapult/+/refs/heads/main/web_page_replay_go/"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="amd64 ~arm ~arm64 ~x86"
+IUSE=""
+
+S=${WORKDIR}/web_page_replay_go/src
```

Comment:
Nit: I usually place this directly under (no blank line between) SRC_URI as they're closely related.